collector_test.go

```diff
@@ -4,13 +4,78 @@ import (
 	"testing"
 
 	"github.com/percona/rds_exporter/config"
+	"github.com/percona/rds_exporter/sessions"
+	"github.com/prometheus/client_golang/prometheus"
 )
 
 func TestNew(t *testing.T) {
+	t.Parallel()
+
+	settings := &config.Settings{}
+	sessions := &sessions.Sessions{}
+	c := New(settings, sessions)
+
+	if c == nil {
+		t.Fatal("collector should not be nil")
+	}
+}
+
+func TestCollector_Collect(t *testing.T) {
```

Comment:
Do not Snake_Case for test methods, that's not a Go style (testing examples are an exception, but that's not an example).
You may want to use subtests for Collect and Describe.

Reply:
> that's not a Go style

Source?

Reply:
https://golang.org/doc/effective_go.html#mixed-caps